doc/source/accumulators.rst

```diff
@@ -34,11 +34,13 @@ Usage of an accumulator/counter::
   a[x]             # get the current value/count for x.
                    # if x was not added to a, it returns zero(V)
 
-  push!(a, x)       # add the value/count for x by 1
-  push!(a, x, v)    # add the value/count for x by v
-  push!(a, a2)      # add all counts from a2 to a1
+  push!(a, x)      # add the value/count for x by 1
+  push!(a, x, v)   # add the value/count for x by v
+  push!(a, a2)     # add all counts from a2 to a1
 
   pop!(a, x)       # remove a key x from a, and returns its current value
 
   merge(a, a2)     # return a new accumulator/counter that combines the
                    # values/counts in both a and a2
+
+most_common(a)     # Return a list of the most common elements and their counts from the most common to the least.
```

Comment:
I think fixing the indentation here should get rid of the weirdness in the rich diff